extensions/hibernate-validator/runtime/src/main/java/io/quarkus/hibernate/validator/runtime/HibernateValidatorBuildTimeConfig.java

```diff
@@ -72,4 +82,19 @@
         @ConfigItem(defaultValue = "false")
         public boolean allowMultipleCascadedValidationOnReturnValues;
     }
+
+    @ConfigGroup
+    public static class HibernateValidatorExpressionLanguageBuildTimeConfig {
+
+        /**
+         * Configure the Expression Language feature level for constraints, allowing to define which
+         * Expression Language features are available for message interpolation.
+         * <p>
+         * This property only affects the EL feature level of "static" constraint violation messages. In particular, it
+         * doesn't affect the default EL feature level for custom violations. Refer to
+         * <code>custom-violation-expression-feature-level</code> to configure that.
```

Comment:
Maybe this would be clearer to the uninitiated?

Also, I think you decided against adding `custom-violation-expression-feature-level`,
so the "Refer to" part is probably no longer relevant?

```suggestion
         * This property only affects the EL feature level of "static" constraint violation messages set through the <code>message</code> attribute of constraint annotations.
         * <p>
         * In particular, it doesn't affect the default EL feature level for custom violations
         * created programmatically in validator implementations.
         * The feature level for those can only be configured directly in the validator implementation.
```